metadata/nz_building_outlines.xml

```diff
@@ -104,48 +104,17 @@
         </gmd:CI_Citation>
       </gmd:citation>
       <gmd:abstract>
-        <gco:CharacterString>This dataset consists of building outlines within mainland New Zealand. This is not a complete set and will be added to as new imagery becomes available.
-
-This dataset includes the spatial coverage of building outlines using remotely sensed information. A building outline is a 2D representation of the roof outline of buildings which have been classified from remotely sensed information using a combination of automated and manual processes to extract and orthogonalise building roof outlines. Structures greater than or equal to 10 square meters are captured in this dataset. Each building polygon represents a building outline and this may include spaces such as decks, garages and porches.
-
-**Source imagery**
-Resolution of the imagery used for capture ranges from 50cm pixel resolution (0.5m GSD) to 7.5cm pixel resolution (0.075m GSD).  Please refer to the NZ Building Outlines Data Dictionary for further information.
-
-**Accuracy Specification:**
-The Building Outlines contain spatial detail reflective of the visible characteristics of building outlines as seen from the source imagery. The building outlines were created by an automated method of feature extraction. 
+        <gco:CharacterString>
+This dataset consists of polygon outlines representing buildings within mainland New Zealand. A building outline is a 2D representation of the roof outline of a building which has been classified from LINZ aerial imagery using a combination of automated and manual processes to extract and orthogonalise a building roof outline. Building roof outlines observed in aerial imagery larger than or equal to 10 square meters are captured in this dataset. Each building polygon represents a building roof outline and this may include structures such as garages and large sheds.
```

Comment:
I have suggested some slight changes to the wording - simplifying the description somewhat to use more plain language terms and reducing the word count:

This dataset provides current outlines of buildings within mainland New Zealand captured from the latest aerial imagery. 

A building outline is a 2D representation of the roof outline of a building which has been classified from LINZ aerial imagery using a combination of automated and manual processes to extract and refine a building roof outline.

Building roof outlines observed in aerial imagery larger than or equal to 10 square meters are captured in this dataset, and may include structures such as garages and large sheds.